docs/io/votable/index.rst

```diff
@@ -414,6 +414,12 @@ VOTable supports a number of different serialization formats.
   supported by the `astropy.io.votable` writer, since it requires
   writing multiple files.
 
+- `PARQUET`
+  stores the data in an external PARQUET file, similar to FITS serialization.
+  Reading and writing is fully supported by the `astropy.io.votable` writer and
+  the `parse()` reader. Works for absolute and relative paths. This serialization
+  can be used by setting the table ``format`` attribute to `votable.parquet`.
+
```

Comment:
Single backticks mean sphinx will try to linkify the words, and will fail as no reference will be found for most of these:
```suggestion
- ``PARQUET``
  stores the data in an external PARQUET file, similar to FITS serialization.
  Reading and writing are fully supported by the `astropy.io.votable` writer and
  the `parse()` reader. Works for absolute and relative paths. This serialization
  can be used by setting the ``'format'`` argument to ``'votable.parquet'``.

```